src/motor-operations.cc

```diff
@@ -84,19 +85,97 @@ static char test_acceleration_ok(float acceleration) {
 }
 #endif
 
+struct history_slot{
```

Comment:
Add a space between type-name and open curly brace.

Regarding type names: I prefer upper case names in the context of this project, so more like HistorySlot.